src/components/Fruit.js

```diff
@@ -0,0 +1,30 @@
+import React, {useState} from "react";
+
+function ChooseFruit({text, fruitName, setFruit}){
+
+    function  setFruitMin (){
+        setFruit(fruitName > 0 ? fruitName-1 : fruitName)
```

Comment:
Om dit gedrag voor elkaar te krijgen zou ik gebruikmaken van het `disabled` attribuut van `button`. Als de waarde `true` is dan zal de onClick niet uitgevoerd worden